cylc/flow/tui/app.py

```diff
@@ -310,11 +310,10 @@ def get_snapshot(self):
                 }
             )
         except WorkflowStopped:
```

Comment:
FYI: I'm going to take a look at getting the client to raise a `WorkflowDoesNotExist` error in place of the WorkflowStopped when appropriate, just need to make sure there aren't any unintended side-effects - see https://github.com/cylc/cylc-flow/pull/4776 (which has a similar problem)

Reply:
This PR is on the back burner. I can close and un-assign the issue since it's right at the bottom of my TODO list if you think that's appropriate.